tests/phpunit/Process/Runner/ParallelProcessRunnerTest.php

```diff
@@ -112,17 +113,17 @@ public static function threadCountProvider(): iterable
         yield 'invalid thread' => [-1];
 
         yield 'nominal' => [4];
-
-        yield 'infinite' => [PHP_INT_MAX];
```

Comment:
Is this something we can no longer test?

Reply:
yes, since now we use `range(1, $threadsCount)`, it leads to 

```
1) Infection\Tests\Process\Runner\ParallelProcessRunnerTest::test_it_handles_all_kids_of_processes_with_infinite_threads with data set "infinite" (9223372036854775807)
ValueError: The supplied range exceeds the maximum array size: start=1 end=9223372036854775807
```

But I've added similar test in the next commit, where threads count is bigger than number of processes